packages/core-data/README.md

```diff
@@ -875,6 +875,47 @@ _Returns_
 
 -   `EntityRecordsResolution< RecordType >`: Entity records data.
 
+### useResourcePermissions
+
+Resolves resource permissions.
```

Comment:
The example is helpful, but we could include more details about what type of permissions it covers, so people have a better overview when scanning the docs.

Reply:
I added a second example, thank you!